### ISSUE TITLE
Weights of the best trained model

### ISSUE THREAD

**Issue by Atmyre** [ZhexinLiang/CLIP-LIT#11]
I also wanted to ask if you could provide weights for the best trained model, results for which you have reported in the paper?

**Comment by ZhexinLiang** [comment]
The enhancement model I used to report in my paper is [pretrained_models/enhancement_model.pth](https://github.com/ZhexinLiang/CLIP-LIT/blob/main/pretrained_models/enhancement_model.pth).

**Comment by Atmyre** [comment]
Thank you for your reply

But I tried to run test.py on this model, and I get quite bad results. Like this: 
<img width="1021" alt="Screenshot 2023-10-28 at 13 16 02" src="https://github.com/ZhexinLiang/CLIP-LIT/assets/9888971/b7790c3c-2b2c-4163-87a7-43730298de9d">

I run the code with this command: python test.py -i ./test_data/BAID/test/resize_input/ -o ./inference_results/ -c ./pretrained_models/enhancement_model.pth

I guess this does not look like pictures from the paper. Could you suggest what I could possibly be doing wrong?

**Comment by Atmyre** [comment]
And the metrics I get on BAID are the following:
PSNR: 9.51
SSIM: 0.61
LPIPS: 0.34

**Comment by ZhexinLiang** [comment]
I check the test data, the checkpoint and the code on this github repo once more. And only find that the test data has some duplicate images (I apologize for not noticing this earlier. I will delete the duplicate ones soon). 

I download the code, the checkpoint and the BAID test data from this repo again and rerun the test experiments. 

For the image you mentioned, I think it is BAID test dataset 0060.JPG. The result I have just generated for 0060.JPG is:
![0060](https://github.com/ZhexinLiang/CLIP-LIT/assets/122451585/d0753130-5955-44f6-8869-4a0a71326774)

The metrics I have just got on BAID test dataset are:
PSNR: 21.611
SSIM: 0.883
LPIPS: 0.159
MUSIQ: 55.683
Even a little bit better than what I repoerted in the paper. The metrics may be changed a little due to different metrics codes, but could never be something like what you mentioned above.

The command you use is correct. Maybe you changed the input test data or the code?
